citi-onboarding/virtu-onepage
# Create Banner section and make it responsive

**What I did:**
- First, I've created the files and imported/exported components
- On Banner.jsx, there's a const text and title - who are responsable to change the content on the page.
- I've created the Banner Section just like the mockup (web and mobile version)

**How to test:**
- Please, open the server at the terminal with
`cd client
yarn start`
- Compare what I have done (web and mobile) with the mockup on Figma
- **please, see the Banner.css and check if there's something I could improve, tips are always welcome!**
- See the responsivity

If you have any issue ou question, please, ask me a review! Thanks :)

resolves #18
